com.unity.robotics.urdf-importer/Runtime/UrdfComponents/UrdfJoints/UrdfJoint.cs

```diff
@@ -274,17 +243,11 @@ public virtual bool AreLimitsCorrect()
 
         protected virtual bool IsJointAxisDefined()
         {
-#if UNITY_2020_1_OR_NEWER
+            // hey Vidur - how can this happen??
```

Comment:
@vidurvij-Unity ^^

Reply:
This is a remnant of ROS#. They had that API for rigid body but I went through the code and I think the axisofMotion is always defined.

Reply:
Right, so we should probably simplify it then since it's always going to be returning true